WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/InitializationManager.java

```diff
@@ -71,6 +79,8 @@ private InitializationManager(Project project) {
         intelliJListener = new IntelliJListener(watchDogEventManager, project);
```

Comment:
Looking at this whole file and with a fresh mind, I think the difference between WatchDogEvents and these other events is not really clear. I'd propose renaming the debug events as such.